tools/clang/unittests/HLSLExec/LongVectorOps.def

```diff
@@ -137,6 +138,10 @@ OP_DEFAULT(Binary, Logical_Or, 2, "or", ",")
 OP(Ternary, Select, 3, "TestSelect", "", " -DFUNC_TEST_SELECT=1",
    "LongVectorOp", SelectCond, Default2, Default3)
 
+OP(Unary, IsFinite, 1, "TestIsFinite", "", " -DFUNC_TEST_ISFINITE=1",
```

Comment:
Is `Unary` the right group to use here?  Why not `UnaryMath` (and if so, should move this up next to the other unary math ops)

Reply:
I suggested to use 'FloatSpecial' as I don't think this is really a 'Math' Op either. 'UnaryLogical' would also fit IMO. But 'FloatSpecial' seems even better.

Reply:
Poor choice of wording on my part. It wasn't that it's not a math op and more so that Math op felt too generic for this context. The trig functions are technically unary math ops as well but grouping them as trig ops felt logical.

Just sharing my thoughts there!